sway/commands/font.c

```diff
@@ -16,12 +17,34 @@ struct cmd_results *cmd_font(int argc, char **argv) {
 	if (strncmp(font, "pango:", 6) == 0) {
 		config->pango_markup = true;
 		config->font = strdup(font + 6);
+		free(font);
```

Comment:
In this branch, I think it makes sense to do:

```c
config->font = font + 6;
free(font[0..5]); // Pseudocode, not sure how to free just the first 5 bytes.
```

This avoids one unecessary `strdup`. Does this make sense? If so, how would you
do this?

Reply:
That's not possible.

But it doesn't matter too much anyways, we shouldn't try micro-optimizing too hard.